src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -489,12 +489,12 @@ def _prune_heads(self, heads_to_prune):
     @replace_return_docstrings(output_type=BaseModelOutput, config_class=_CONFIG_FOR_DOC)
```

Comment:
```suggestion
    @replace_return_docstrings(output_type=BaseModelOutputWithPooling, config_class=_CONFIG_FOR_DOC)
```

Reply:
This isn't your fault again, but I notice the docstring decorator has the wrong type here - your type hints are correct!

Reply:
Interesting, I'll try to catch these guys in the future